src/components/ResourceTypeSelector/ResourceTypeSelector.js

```diff
@@ -20,7 +20,7 @@ const CategoryButton = ({ resourceType, selectedResourceType, selectResourceType
     === resourceType ? styles.buttonSelectedText : styles.buttonText;
 
   return (
-    <Button style={buttonStyle} onPress={() => selectResourceTypeAction(resourceType)}>
+    <Button rounded style={buttonStyle} onPress={() => selectResourceTypeAction(resourceType)}>
       <Text style={buttonTextStyle}>{categoryDisplay}</Text>
     </Button>
```

Comment:
This isn't the best fix. I tested this below and it works:
`<TouchableOpacity style={buttonStyle} onPress={() => selectResourceTypeAction(resourceType)}>
      <Text style={buttonTextStyle}>{categoryDisplay}</Text>
    </TouchableOpacity>`
 
 Also need style changes to `button` and `buttonSelected`: add `padding: 10` and `borderRadius: 20` to both.

Reply:
What's wrong with it? Seems very nice and clean, and defers responsibility to the framework.

Reply:
The onPress ripple area is still different than the body of the button.